release.sh

```diff
@@ -162,6 +168,10 @@ mkdir -p target/tmp || echo "target/tmp exists"
 i=1; until curl -XGET http://${DOCKER_IP}:19999/arlas/swagger.json -o target/tmp/swagger.json; do if [ $i -lt 60 ]; then sleep 1; else break; fi; i=$(($i + 1)); done
 i=1; until curl -XGET http://${DOCKER_IP}:19999/arlas/swagger.yaml -o target/tmp/swagger.yaml; do if [ $i -lt 60 ]; then sleep 1; else break; fi; i=$(($i + 1)); done
 
+mkdir -p arlas-spec
```

Comment:
Maybe this folder would better be called `openapi` (no need to prefix with `arlas-` since it is located in arlas-server repository).

Reply:
done :)